### PR TITLE
Added documentation for the umbLayoutSelector selector

### PR DESCRIPTION
**Pull request by abjerner**
As pointed out by a user the other day, the `umbLayoutSelector` wasn't documented ... so I thought I'd do something about that 😄

### CODE REVIEW

File: Reference/Angular/Directives/umbLayoutSelector/Index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# umbLayoutSelector
```
Comment (sofietoft) [inline Reference/Angular/Directives/umbLayoutSelector/Index.md:1]:
```suggestion
---
versionFrom: 8.0.0
---


# umbLayoutSelector
```

File: Reference/Angular/Directives/umbLayoutSelector/Index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+When your have a list of items or similar, you can use the `umb-layout-selector` directive to let users toggle between different layouts. For instance in Umbraco's media archive, users can select between a grid based layout (thumbnails) and a list based layout (table).
```
Comment (sofietoft) [inline Reference/Angular/Directives/umbLayoutSelector/Index.md:3]:
```suggestion
When you have a list of items or similar, you can use the `umb-layout-selector` directive to let users toggle between different layouts. For instance, in Umbraco's media archive, users can select between a grid-based layout (thumbnails) and a list-based layout (table).
```

File: Reference/Angular/Directives/umbLayoutSelector/Index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        layout.active = true;
+    };
+
+    // Select the first layout
+    $scope.selectLayout($scope.layouts[0]);
+
+});
+```
+
+For each layout, the `name` property indicates the visual name of the layout (eg. used when hovering over the layout in the selector), and `icon` is the CSS selector for the icon of the layout. `path` works a sort of alias, and is used internally for comparing the layouts. And while a bit misleading, each layout should also have a `selected` property indicating whether a particular layout is enabled, and thereby visible in the selector.
```
Comment (sofietoft) [inline Reference/Angular/Directives/umbLayoutSelector/Index.md:61]:
```suggestion
For each layout, the `name` property indicates the visual name of the layout (eg. used when hovering over the layout in the selector), and `icon` is the CSS selector for the icon of the layout. The `path` value works a sort of alias, and is used internally for comparing the layouts. And while a bit misleading, each layout should also have a `selected` property indicating whether a particular layout is enabled, and thereby visible in the selector.
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
Looks great @abjerner !

Thanks a bunch or adding the article 💪 
I've made a few suggestions - mind taking a look?

File: Reference/Angular/Directives/umbLayoutSelector/Index.md (hunk 1 above)
Comment (abjerner) [reply Reference/Angular/Directives/umbLayoutSelector/Index.md:1]: My bad. I should have remembered 😊

File: Reference/Angular/Directives/umbLayoutSelector/Index.md (hunk 3 above)
Comment (abjerner) [reply Reference/Angular/Directives/umbLayoutSelector/Index.md:66]: Would this be better?

> The `path` attribute indicates a sort

Comment (sofietoft) [reply Reference/Angular/Directives/umbLayoutSelector/Index.md:66]: Yes! That's a much better solution - Thanks! 😁

Comment (abjerner) [reply Reference/Angular/Directives/umbLayoutSelector/Index.md]: Should be fixed now 😉

**Review by sofietoft — APPROVED** [review]
Beautiful @abjerner ! Thanks a lot for adding this article.

Let's get it merged in 💪

**Pull request merged**